### PR TITLE
University of Wisconsin-Madison Libraries Catalog.js -- fix eslint wa…

### PR DESCRIPTION
**Pull request by retorquere**
…rnings

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: University of Wisconsin-Madison Libraries Catalog.js [hunk 1]
```diff
@@ -59,20 +60,21 @@ function detectWeb(doc, url) {
 				return "computerProgramm";
 			case "Photos, Drawings, Prints":
 				return "artwork";
-			//case "Journals, Magazines, Newspapers":
-			//there is no such itemType yet
+				// case "Journals, Magazines, Newspapers":
+			// there is no such itemType yet
```
Comment (zuphilip) [inline University of Wisconsin-Madison Libraries Catalog.js]: No, this intending is not what was mean.

I suggest to move this case just before the `default` case as
```js
case "Journals, Magazines, Newspapers": // there is no such itemType yet
```

Comment (retorquere) [reply University of Wisconsin-Madison Libraries Catalog.js]: Done. But if there's a `default` anyway, why have the two fall-through cases above it?

Comment (zuphilip) [reply University of Wisconsin-Madison Libraries Catalog.js]: If in the future other itemTypes for these cases will exists, then we can change the behavior here easily.

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**